README-mc.md

```diff
@@ -0,0 +1,52 @@
+# Firefox Home (New Tab)
+
+This directory contains some source files (jsx, sass) that need to be built outside of the `./mach` build system.
+We are working on migrating this build system to work with `mach`. In the meantime, please follow the following
+steps if you need to make changes to code in this directory:
```

Comment:
Maybe a note that the commands are run from m-c root directory where one would normally run `./mach` And perhaps a note of going directly into `browser/components/newtab` to skip the `--prefix …`